wallet/src/de/schildbach/wallet/ui/SetPinActivity.kt

```diff
@@ -455,8 +459,13 @@ class SetPinActivity : InteractionAwareActivity() {
     }
 
     private fun startVerifySeedActivity() {
-        val intent = VerifySeedActivity.createIntent(this, seed.toTypedArray())
-        startActivityNewTask(intent)
+        val onboardingInvite = intent.getBooleanExtra(EXTRA_ONBOARDING_INVITE, false)
+        val verifySeedActivityIntent = VerifySeedActivity.createIntent(this, seed.toTypedArray())
+        if (onboardingInvite) {
+            startActivityNewTask(OnboardFromInvite.createIntent(this, OnboardFromInvite.Mode.STEP_3, verifySeedActivityIntent))
+        } else {
+            startActivityNewTask(verifySeedActivityIntent)
+        }
```

Comment:
Some of these changes may conflict with the current strandhogg merge PR.  We will just have to see how it turns out after this PR is merged.